cli/cobra.go

```diff
@@ -166,6 +166,7 @@ func (tcmd *TopLevelCommand) HandleGlobalFlags() (*cobra.Command, []string, erro
 	// arguments are in `flags.Args()`.
 	if err := flags.Parse(tcmd.args); err != nil {
 		// Our FlagErrorFunc uses the cli, make sure it is initialized
+		// TODO(thaJeztah): does this need OTEL global tracer and meter to be configured?
```

Comment:
Oh! Can someone have a quick peek here? Do I need to keep this TODO or safe to remove?

Reply:
the global providers _shouldn't_ be necessary here

Reply:
Thx! Removed